### PR TITLE
Completed css-two-column-layout,

### PR DESCRIPTION
**Pull request by Alex-Davis0**
Screenshot 2021-12-02 192513

### CODE REVIEW

File: css-two-column-layout/styles.css [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+.row {
+  display: flex;
+  justify-content: center;
+  align-content: center;
+  letter-spacing: 0.1rem;
+  line-height: 1.5rem;
+}
+
+.column-one-forth {
```
Comment (BrettAlbright) [inline css-two-column-layout/styles.css:14]: make sure your column classes have actual width set on them. Right now flexbox is controlling the column size and you don't want that :)

File: css-two-column-layout/styles.css [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  margin: 1.5rem;
+}
+
+.column-one-half {
```
Comment (BrettAlbright) [inline css-two-column-layout/styles.css]: same as above

File: css-two-column-layout/styles.css [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+  max-width: 1050px;
+  margin-left: auto;
+  margin-right: auto;
+  font-family: sans-serif;
+}
+
+.row {
+  display: flex;
+  justify-content: center;
+  align-content: center;
```
Comment (BrettAlbright) [inline css-two-column-layout/styles.css]: I would move these out of the row class and create another class such as `justify-center` and `align-center` so you can apply those classes individually. A `row` class is typically just a horizontal section of the page.

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
Looking nice, a few updates till complete!

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**